release-gap-package

```diff
@@ -412,7 +412,8 @@ cd "$TMP_DIR/$BASENAME"
 notice "Removing unnecessary files"
 # Remove recursively in case there is a .github directory
 rm -rf .git* .hg* .cvs* .circleci
-rm -f .appveyor.yml .codecov.yml .travis.yml
+rm -f .appveyor.yml .codecov.yml .travis.yml azure-pipelines.yml
+rm -f .DS_Store
```

Comment:
In fact maybe it's worth deleting any of these files from subfolders too - does anyone know how to do that? Would this do the job?
```suggestion
rm -f **/.DS_Store
```

Reply:
That would work in bash, but the script is meant to work in arbitrary shells (including `dash`, which Debian uses), so it probably won't work. We could perhaps use the `find` tool, but this, too, has different behavior in different version. But we could just say we require POSIX, and follow what is available as per [the Open Group Base Specifications](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/find.html). So this might work:
```suggestion
find . -name .DS_Store -exec rm -f {} +
```

Reply:
Works on my computer at least. Pushed.